js/plugins/vertexai/src/gemini.ts

```diff
@@ -53,8 +53,21 @@ const SafetySettingsSchema = z.object({
   threshold: z.nativeEnum(HarmBlockThreshold),
 });
 
+const VertexRetrievalSchema = z.object({
+  datastore: z.object({
+    projectId: z.string(),
+    location: z.string(),
```

Comment:
can you make these optional? can default to the project/location that we use for the vertex client: https://github.com/firebase/genkit/blob/main/js/plugins/vertexai/src/index.ts#L105-L106

Reply:
done